test/any/web_apis/web_apis-integration.es6.js

```diff
@@ -29,13 +29,15 @@ describe('WebAPI and api extractor', function() {
     let edge14 = global.DATA.edge14;
     let safari602 = global.DATA.safari602;
     let og = global.ObjectGraph;
-    let extractor = org.chromium.apis.web.apiExtractor.create({});
-    let apiImporter = org.chromium.apis.web.ApiImporter.create();
+    // Construct service components in context:
```

Comment:
@jingt06 this is not actually the right solution. We should have the three DAOs exported by ApiMatrix (same names as ApiImporter). They run on different machines, so relying on contextualization to get DAOs in the Matrix to work is not a viable strategy.